build.gradle.kts

```diff
@@ -186,6 +189,9 @@ tasks {
         useJUnitPlatform()
         maxParallelForks = 8
     }
+    withType<Test>().matching { it.name.startsWith("compatTest") }.configureEach {
+        systemProperty("plugin.version", project.version)
```

Comment:
We can probably set for all the tests, not just `compatTest` (to reuse the previous section in `build.gradle`).

Reply:
I didn't want to add an unnecessary input to the other test tasks.